src/app/shared/entry/verified-display/verified-display.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="row">
+  <mat-expansion-panel [expanded]="true">
+    <mat-expansion-panel-header>
+      <mat-panel-title>
+        Verification Status
+      </mat-panel-title>
+    </mat-expansion-panel-header>
+    <table mat-table [dataSource]="dataSource" class="mat-elevation-z4" style="width: 100%" matSort>
+      <ng-container matColumnDef="path">
+        <mat-header-cell *matHeaderCellDef mat-sort-header> Test Parameter File </mat-header-cell>
+        <mat-cell  *matCellDef="let element"> {{element.path}} </mat-cell>
+      </ng-container>
+      <ng-container matColumnDef="platform">
+        <mat-header-cell *matHeaderCellDef mat-sort-header> Platform </mat-header-cell>
+        <mat-cell *matCellDef="let element"> {{element.platform}} </mat-cell>
+      </ng-container>
+      <ng-container matColumnDef="metadata">
```

Comment:
I think I prefer `Verifier` from the specification

Reply:
"metadata" is not user-facing, but I'll change it anyways for consistency.

Reply:
Oh, I meant `Verified Source` => `Verifier` but its good to keep the two consistent